internal/command/checks/command.go

```diff
@@ -0,0 +1,90 @@
+package checks
+
+import (
+	"fmt"
+	"io"
+	"os"
+	"strings"
+
+	"github.com/pkg/errors"
+	"github.com/spf13/cobra"
+	"golang.stackrox.io/kube-linter/internal/builtinchecks"
+	"golang.stackrox.io/kube-linter/internal/check"
+	"golang.stackrox.io/kube-linter/internal/command/common"
+	"golang.stackrox.io/kube-linter/internal/defaultchecks"
+	"golang.stackrox.io/kube-linter/internal/ternary"
+)
+
+var (
+	dashes = func() string {
+		var sb strings.Builder
+		for i := 0; i < 30; i++ {
+			sb.WriteRune('-')
+		}
+		return sb.String()
+	}()
+
+	formatsToRenderFuncs = map[string]func([]check.Check, io.Writer){
+		common.PlainFormat: func(checks []check.Check, out io.Writer) {
```

Comment:
Move these to separate functions instead of inlining them. I would also encourage you to use Go templates for formatting. Especially the markdown formatting (a) looks ugly in code and (b) is extremely limited as you cannot add any custom styling, notes, etc.
(You might need to add either a custom function or translate the checks into a separate type to realize the "enabled by default" flag)

Reply:
Done. Also moved to templates.